apps/rush/src/actions/RushCommandLineParser.ts

```diff
@@ -90,6 +92,15 @@ export default class RushCommandLineParser extends CommandLineParser {
     }
   }
 
+  private _ensureEnvironment(): void {
+    /* tslint:disable-next-line:no-string-literal */
+    let environmentPath: string = process.env['PATH'];
+    environmentPath = path.join(this.rushConfiguration.commonTempFolder, 'node_modules', '.bin') +
```

Comment:
One of the versions will end up as top-level dependency (since npm v3). So one of them will end up in the .bin folder. 

---
In reply to: [119751422](https://github.com/Microsoft/web-build-tools/pull/218#discussion_r119751422) [](ancestors = 119751422)